test/parallel_api/iterator/zip_iterator.pass.cpp

```diff
@@ -614,7 +614,7 @@ DEFINE_TEST(test_merge)
         retrieve_data(host_keys, host_vals, host_res_merge);
 
         auto host_first1 = host_keys.get();
-        auto host_first2 = host_vals.get();
+        [[maybe_unused]] auto host_first2 = host_vals.get();
```

Comment:
I mean it's not maybe_unused. It's unused. What is the purpose to keep this line?

Reply:
Fixed.